app/models/iscsi_address.rb

```diff
@@ -0,0 +1,5 @@
+class IscsiAddress < Address
+  def get_address_string()
+    "iscsi"
```

Comment:
Return the value of the iqn field, not "iscsi" string.

Reply:
done